Solutions/ImpervaCloudWAF/Data Connectors/ImpervaWAFCloudSentinelConnector/__init__.py

```diff
@@ -156,6 +156,8 @@ def parse_cef(self,cef_raw):
                 parsed_cef[(parsed_cef[f'{elem}Label']).replace(" ", "")] = parsed_cef[elem]
                 parsed_cef.pop(f'{elem}Label')
                 parsed_cef.pop(elem)
+        if 'start' in parsed_cef.keys() and parsed_cef['start'] is not None and parsed_cef['start']!="":
+            parsed_cef['EventGeneratedTime'] = datetime.datetime.utcfromtimestamp(int(parsed_cef['start'])/1000.0).isoformat()

```

Comment:
Does it make sense to have in try {} catch {}, because in case of any parse issues / errors, we still want to proceed. So that we will assign Default time for TimeGenerated